molecule/Dockerfile.j2

```diff
@@ -25,7 +25,7 @@ gpgkey=https://packages.confluent.io/clients/rpm/archive.key\n\
 enabled=1\n\
 \n\
 [Confluent]\n\
-name=Confluent repository\n\
+name=Confluent repository\n\roles/common/tasks/debian.yml
```

Comment:
Why are we adding this? roles/common/tasks/debian.yml

Reply:
Sorry, it was a mistake